docs/reference/setup/install/docker.asciidoc

```diff
@@ -48,7 +48,7 @@ Elasticsearch can be quickly started for development or testing use with the fol
 
 ["source","sh",subs="attributes"]
 --------------------------------------------
-docker run -p 9200:9200 -e "http.host=0.0.0.0" -e "transport.host=127.0.0.1" {docker-image}
+docker run -p 9200:9200 {docker-image} -E "http.bind_host=0.0.0.0" -E "http.publish_host=127.0.0.1" -E "transport.host=127.0.0.1"
```

Comment:
This won't work as we don't define an ENTRYPOINT in the Dockerfile.

e.g.

```shell
$ docker run -p 9200:9200 docker.elastic.co/elasticsearch/elasticsearch:5.1.1 -E "http.bind_host=0.0.0.0" -E "http.publish_host=127.0.0.1" -E "transport.host=127.0.0.1"
container_linux.go:247: starting container process caused "exec: \"-E\": executable file not found in $PATH"
docker: Error response from daemon: invalid header field value "oci runtime error: container_linux.go:247: starting container process caused \"exec: \\\"-E\\\": executable file not found in $PATH\"\n".
```

See also the general comment.